packages/app/src-tauri/Cargo.toml

```diff
@@ -41,7 +40,7 @@ tempfile = "3.8.1"
 thiserror = "1.0.56"
 tokio = { version = "1", features = ["fs", "macros"] }
 tokio-util = {version = "0.7.10", features = ["compat"] }
-uuid = {version = "1.6.1", features = ["v7", "fast-rng", "serde",] }
+tauri-plugin-channel = { git = "https://github.com/fourviere/tauri-plugin-channel.git", branch = "sender_receiver_ts" }
```

Comment:
Can we release the project as a package without passing it through git?

Reply:
Of course, we just need to create tokens for npm and crates.io and setup the covector action on the plugin repo